File: app/Models/Notificacion.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Notificacion extends Model
{
    use HasFactory;

    protected $table = 'comentarios';

    protected $fillable =
        [
            'id',
            'titulo',
            'descripcion',
            'fecha_envio',
            'fecha_visto',
            'id_cliente',
            'created_at',
            'updated_at'
        ];
}

File: models/notificacion.go
package models

import "time"

type Notificacion struct {
	ID          uint       `json:"id" gorm:"primaryKey"`
	Titulo      string     `json:"titulo"`
	Descripcion string     `json:"descripcion"`
	FechaEnvio  time.Time  `json:"fecha_envio"`
	FechaVisto  *time.Time `json:"fecha_visto,omitempty"`
	IDCliente   uint       `json:"id_cliente"`
	CreatedAt   time.Time  `json:"created_at"`
	UpdatedAt   time.Time  `json:"updated_at"`
}

func (Notificacion) TableName() string {
	return "notificaciones"
}
